### ISSUE TITLE
Run flutter packages get for all the projects.

### ISSUE THREAD

**Issue by feinstein** [brianegan/flutter_architecture_samples#114]
I don't know if this is even possible, but it will be nice to have a way to run `flutter packages get` only once and have all the projects dependencies satisfied at once.

**Comment by mmcc007** [comment]
There's a script for that:
```
scripts/ci.sh --get
```

**Comment by feinstein** [comment]
Interesting, this should be on the Readme.md (if it is I missed it completely)

**Comment by mmcc007** [comment]
Well, I suppose it was not anticipated it would come up... but now that you mention it.. it should probably be in there.

**Comment by george2seven** [comment]
Hello,

I am trying to run this script on Windows 10 and I am getting the following error:

`Cannot read link, path = [folder name] (OS Error: The file or directory is not a reparse point.
, errno = 4390)
Running "flutter pub get" in redux...
pub get failed (66; , errno = 4390))`

After digging around a bit the problem seems to be with the pubspec.yaml referencing other parts of the app and Windows (or flutter ?) not been able to complete the task or fine the path(?).

I've also tried running flutter package get in individual folders and the problem repeats in the ones that have reference in other folders in the pubspec.yaml.

Can you please provide some assistance as to how I can run the project on Windows 10?

Thank you very much

**Comment by brianegan** [comment]
Oh boy, sorry about that @george2seven -- I'll admit I never use Windows and kinda assumed the Flutter / Dart tools would handle `/` vs `\` in yaml files properly for us :/

I'll rig up my Windows 10 gaming machine with a Flutter environment to see if I can get this one fixed up. Thanks for reporting :)

**Comment by george2seven** [comment]
Thank you very much @brianegan !

**Comment by ride4sun** [comment]
what is the status of this?

**Comment by feinstein** [comment]
The Flutter tool has something similar for getting all the dependencies of all the Flutter git project, maybe this can be adapted? I never tried, but maybe it's compatible?

**Comment by ride4sun** [comment]
@feinstein  can you elaborate?

**Comment by feinstein** [comment]
Check the Flutter repo for "Contributing", there they explain how to setup your environment, download the code and etc. One of the steps is to use the Flutter tool to fetch all the dependencies of all the inner projects with their respectives YAML. I at my phone right now, otherwise I would point it out. If you can't find it, please let me know.

**Comment by ride4sun** [comment]
Sorry looked into the repos. but could not find what your are talking about.

**Comment by feinstein** [comment]
Take a look at [here] (https://github.com/flutter/flutter/wiki/Tree-hygiene), it makes a reference to:

```
flutter update-packages
```

**Comment by fzyzcjy** [comment]
This is a very useful script! I personally have multiple packages in my own project, and this script helps me to bulk `flutter pub get` :)

**Comment by Rexios80** [comment]
I made https://pub.dev/packages/puby which works in all flutter/dart projects and mono-repos

**Comment by NikitaRomanenko** [comment]
@Rexios80 nice script. I wonder if someone create a plugin for android studio
